src/reduceHandlers.js

```diff
@@ -1,33 +1,66 @@
 export const someFatalHandlers = (...handlers) => {
   return (error, action) => {
-    let data
-    handlers.some((handler) => {
-      data = handler(error, action)
-      return typeof data !== 'undefined'
-    })
-    return data
+    return handlers.reduce((data, handler) => {
+      if (typeof data !== 'undefined') {
+        if (isPromise(data)) {
+          return data
+            .catch((rejection) => {
+              error = rejection
```

Comment:
The old behavior was that if one of the handlers threw, none of the subsequent handlers would be called. I think the new behavior (calling subsequent handlers with the new error) is more intuitive.